### DISCUSSION TITLE
TinyMCE does not work well

### DISCUSSION THREAD

**Discussion by PH5HP** [e107inc/e107#5327] [General]
Hi,

I am wondering what is happening with TinyMCE. I tried to EDIT one of my pages I created a while ago, but every time I save it the changes I made are not there. This is what I did:

- Edit the page
- Go to the source window
- Made my changes in the source of the page
- Click on the OK button
- Save the page

Every time I do that the changes are apparently not saved and the page remains how it was before I tried to EDIT the page. 

Kind regards,
Henk - https://ph5hp.nl

**Comment by Jimmi08** [comment]
Hi, e107 version?  Did you check console for javascript errors? Are changes saved in the normal way (without the source code button)?

**Reply by PH5HP** [reply]
Hi Jimmi08,

All valid questions.

- Latest E107 version.
- I am not sure, but most of the time it is saved in the normal way (without the source code button).

I found these javascript messages when using TinyMCE in source code mode.
![image](https://github.com/user-attachments/assets/a7ad28da-76ae-4981-853a-b2965433d316)

Kind regards,
Henk - https://ph5hp.nl/
